common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -622,14 +626,6 @@ private static boolean hasUnsafe0() {
         }
     }
 
-    private static long arrayBaseOffset0() {
-        if (!hasUnsafe()) {
-            return -1;
-        }
-
-        return PlatformDependent0.arrayBaseOffset();
```

Comment:
@normanmaurer oh got ya, yes this what I am doing anyway here https://github.com/netty/netty/pull/3626/files#diff-8d83d75ebf8a18cc48bf0a0b1183c188R116. So it will be -1 if there is no unsafe available, just like before. WDYT?